docs/manual/apk_extensions.rst

```diff
@@ -0,0 +1,160 @@
+.. _manual_apk_extensions:
+
+APK Extensions
+==============
+
+Mobile and XR operating systems and app distribution platforms commonly impose strict size limits on apps,
+especially compared to desktop platforms. For example, apps on
+`Meta Horizon are limited to 1 GB in size <https://developers.meta.com/horizon/resources/vrc-quest-packaging-5/>`_
+for the APK file. While not necessarily a problem for some apps, games often require considerable
+assets and can easily exceed those limits.
+
+Thankfully, there are ways to distribute additional data beyond the single APK file: APK extensions.
+*Meta Horizon* supports two kinds of APK extensions, as outlined
+`in the documentation <https://developers.meta.com/horizon/documentation/native/ps-assets/>`_:
+**Opaque Binary Blob (OBB)** files and **Required Asset** files.
+
+Opaque Binary Blobs (OBBs) vs. Required Assets
+----------------------------------------------
+
+**Opaque Binary Blobs (OBBs)** were supported in Android starting with version
+`2.3 Gingerbread (2010) <https://en.wikipedia.org/wiki/Opaque_binary_blob>`_.
+**Required Asset** files are specific to *Meta Horizon*.
+They are similar, but different (see the
+`documentation <https://developers.meta.com/horizon/documentation/native/ps-assets/>`_ for full details).
+
+Both allow distributing arbitrary data of up to 4 GB per file (although 2 GB are recommended),
+both are located at a similar path in the system and need to be checked for (and downloaded if missing) by
+the app at runtime, both may be installed automatically when the app is installed.
+
+While both support arbitrary file types, **OBB**s *can* be special ZIP archives (with ``.obb`` extension) that support
+signing and encryption (built via Android's ``jobb`` tool).
+However, official Android documentation and tooling for this seems to be in the process of being phased out.
+Keep in mind that these special OBB files may have performance implications when streaming media from them or
+accessing other assets, as they have to be uncompressed into memory first. Consider either storing assets requiring
+fast access uncompressed or as standalone expansion files instead of the special OBB compressed file format.
+For example, you could use a Godot PCK file directly instead.
+
+Note that noll at distribution platforms support all formats or impose the same limits. **Required Asset** files are
+specific to *Meta Horizon*, and *Google Play* has lower limits for OBB expansion files (capping them at 2 GB).
```

Comment:
Google Play no longer support obb expansion files for new apps; see https://developer.android.com/guide/playcore/asset-delivery